jenkins/image_building/upload-node-image.sh

```diff
@@ -68,3 +68,7 @@ upload_node_image() {
         echo "${MAPFILE[i]} has been deleted!"
     done
 }
+
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
```

Comment:
This is unintuitive for readers, put a comment there what it does and when it is triggered

Reply:
Thanks. I thought that was a common practice, but I could be wrong.